src/backend/vulkan/PipelineLayoutVk.cpp

```diff
@@ -22,8 +22,8 @@
 
 namespace backend { namespace vulkan {
 
-    PipelineLayout::PipelineLayout(PipelineLayoutBuilder* builder)
-        : PipelineLayoutBase(builder), mDevice(ToBackend(builder->GetDevice())) {
+    PipelineLayout::PipelineLayout(Device* device, const nxt::PipelineLayoutDescriptor* descriptor)
+        : PipelineLayoutBase(device, descriptor), mDevice(device) {
```

Comment:
Since PipelineLayout only seems to need the FencedDeleter from the Device, I wonder if it should grab that here and stash it in a member instead. Just a thought.

Reply:
Not sure. It seems slightly simpler to keep it this way - it's more like other classes that save an mDevice, and it's simpler in case in the future we need something else from the Device.

Reply:
OK, that's cool.